src/Utilities/functions.jl

```diff
@@ -251,23 +229,22 @@ function test_models_equal(model1::MOI.ModelLike, model2::MOI.ModelLike, variabl
         f2 = MOI.get(model2, MOI.ConstraintFunction(), index2)
         s1 = MOI.get(model1, MOI.ConstraintSet(), index1)
         s2 = MOI.get(model2, MOI.ConstraintSet(), index2)
-        @test isapprox(f1, map_variables(f2, variablemap_2to1))
+        @test isapprox(f1, mapvariables(variablemap_2to1, f2))
```

Comment:
Good catch ! Why was this working ? Insufficient testing ?

Reply:
I see, this is related to https://github.com/JuliaOpt/MathOptInterface.jl/issues/422. You can add `Closes #422` in the comment of this PR then.